src/Symfony/Component/Translation/Bridge/Loco/Tests/LocoProviderTest.php

```diff
@@ -128,89 +107,116 @@ public function testCompleteWriteProcess()
                 $this->assertEquals('https://localise.biz/api/tags.json', $url);
                 $this->assertEquals($expectedAuthHeader, $options['normalized_headers']['authorization'][0]);
                 $this->assertEquals(http_build_query(['name' => 'validators']), $options['body']);
 
-                return $createTagResponse;
+                return new MockResponse('', ['http_code' => 201]);
             },
-            'tagAsset2' => function (string $method, string $url, array $options = []) use ($tagAssetResponse, $expectedAuthHeader): ResponseInterface {
+            'tagAsset2' => function (string $method, string $url, array $options = []) use ($expectedAuthHeader): ResponseInterface {
                 $this->assertEquals('POST', $method);
                 $this->assertEquals('https://localise.biz/api/tags/validators.json', $url);
                 $this->assertEquals($expectedAuthHeader, $options['normalized_headers']['authorization'][0]);
-                $this->assertEquals('post.num_comments', $options['body']);
+                $this->assertEquals('1234', $options['body']);
 
-                return $tagAssetResponse;
+                return new MockResponse();
             },
-
-            'getLocales1' => function (string $method, string $url, array $options = []) use ($getLocalesResponse, $expectedAuthHeader): ResponseInterface {
+            'getLocales1' => function (string $method, string $url, array $options = []) use ($expectedAuthHeader): ResponseInterface {
                 $this->assertEquals('GET', $method);
                 $this->assertEquals('https://localise.biz/api/locales', $url);
                 $this->assertEquals($expectedAuthHeader, $options['normalized_headers']['authorization'][0]);
 
-                return $getLocalesResponse;
+                return new MockResponse('[{"code":"en"}]');
             },
+            'getAssetsIds1' => function (string $method, string $url, array $options = []) use ($expectedAuthHeader): ResponseInterface {
+                $this->assertEquals('GET', $method);
+                $this->assertEquals('https://localise.biz/api/assets?filter=messages', $url);
+                $this->assertEquals(['filter' => 'messages'], $options['query']);
+                $this->assertEquals($expectedAuthHeader, $options['normalized_headers']['authorization'][0]);
 
-            'translateAsset1' => function (string $method, string $url, array $options = []) use ($translateAssetResponse, $expectedAuthHeader): ResponseInterface {
+                return new MockResponse('[{"id":"1337"}]');
+            },
+            'translateAsset1' => function (string $method, string $url, array $options = []) use ($expectedAuthHeader): ResponseInterface {
                 $this->assertEquals('POST', $method);
-                $this->assertEquals('https://localise.biz/api/translations/a/en', $url);
+                $this->assertEquals('https://localise.biz/api/translations/1337/en', $url);
```

Comment:
Can you use assertSame where possible?

Reply:
I've replaced `assertEquals` with `assertSame` for all string and array comparisons. I've kept `assertEquals` for object comparisons. Am I right? ;)

Reply:
👍